src/System.Globalization/tests/CompareInfo/CompareInfoTests.IndexOf.cs

```diff
@@ -63,30 +61,6 @@ public static IEnumerable<object[]> IndexOf_TestData()
             yield return new object[] { s_invariantCompare, "cbabababdbaba", "ab", 0, 13, CompareOptions.None, 2 };
         }
 
-        public static IEnumerable<object[]> IndexOf_Random_TestData()
-        {
-            string[] interestingStrings = new string[] { "", "a", "1", "-", "A", "!", "abc", "aBc", "a\u0400Bc", "I", "i", "\u0130", "\u0131", "A", "\uFF21", "\uFE57" };
```

Comment:
Yes, please do.  Some of those values have special meaning, e.g. I suspect "I", "i", 0x130, and 0x131 were all chosen due to relevance to Turkish casing.  One entry for each of the special characters if they don't already exist should be sufficient.